lib/msal-browser/src/cache/DatabaseStorage.ts

```diff
@@ -110,4 +110,28 @@ export class DatabaseStorage<T>{
             dbPut.addEventListener("error", e => reject(e));
         });
     }
+
+    /**
+     * Remove all records from the database
+     */
+    async clear(): Promise<void> {
```

Comment:
Today, we don't use any accounts hints for storing things in indexeddb, maybe we should? That way we can properly clear records if the the app provides an account to the logout function.

Reply:
One easy way to do this would be to create separate DB's for each user.

Reply:
I think it would be useful. At the moment we only use the `req_cnf` claim (for the keys at least). Maybe we can think up a cache key naming scheme like we have for artifacts cached in localStorage.